build.gradle

```diff
@@ -10,7 +10,8 @@ plugins {
   id 'org.checkerframework' version '0.5.16' apply false
   id 'org.jetbrains.grammarkit' version '2020.3.2' apply false
   id 'org.jetbrains.intellij' version '0.7.2' apply false
-  // Minor Kotlin version is effectively enforced by the IntelliJ version we're building against
+  // Minor Kotlin version is effectively enforced by the IntelliJ version we're building against;
+  // e.g. 2020.x uses version 1.3.x amd 2021.1 uses 1.4.x
```

Comment:
Hmm not good.... plugin `0.9.1-4-SNAPSHOT+git.f14b094` against IJ 2020.3.2....

```
kotlin.jvm.KotlinReflectionNotSupportedError: Kotlin reflection implementation is not found at runtime. Make sure you have kotlin-reflect.jar in the classpath
	at kotlin.jvm.internal.CallableReference.getReflected(CallableReference.java:98)
	at kotlin.jvm.internal.PropertyReference.getReflected(PropertyReference.java:31)
	at kotlin.jvm.internal.MutablePropertyReference0.getGetter(MutablePropertyReference0.java:40)
	at com.intellij.ui.layout.CellKt.createPropertyBinding(Cell.kt:81)
	at com.intellij.ui.layout.Cell.checkBox(Cell.kt:671)
	at com.intellij.ui.layout.Cell.checkBox$default(Cell.kt:275)
	at com.virtuslab.gitmachete.frontend.actions.dialogs.DeleteBranchOnSlideOutSuggestionDialog$createCenterPanel$$inlined$panel$lambda$2.invoke(DeleteBranchOnSlideOutSuggestionDialog.kt:78)
	at com.virtuslab.gitmachete.frontend.actions.dialogs.DeleteBranchOnSlideOutSuggestionDialog$createCenterPanel$$inlined$panel$lambda$2.invoke(DeleteBranchOnSlideOutSuggestionDialog.kt:18)
	at com.intellij.ui.layout.RowBuilder$DefaultImpls.row(Row.kt:42)
	at com.intellij.ui.layout.Row.row(Row.kt:107)
	at com.intellij.ui.layout.LayoutBuilder.row(LayoutBuilder.kt)
	at com.intellij.ui.layout.RowBuilder$DefaultImpls.row$default(Row.kt:41)
	at com.virtuslab.gitmachete.frontend.actions.dialogs.DeleteBranchOnSlideOutSuggestionDialog.createCenterPanel(DeleteBranchOnSlideOutSuggestionDialog.kt:77)
	at com.virtuslab.gitmachete.frontend.actions.dialogs.DeleteBranchOnSlideOutSuggestionDialog.createCenterPanel(DeleteBranchOnSlideOutSuggestionDialog.kt:18)
	at com.intellij.openapi.ui.DialogWrapper.init(DialogWrapper.java:1331)
	at com.virtuslab.gitmachete.frontend.actions.dialogs.DeleteBranchOnSlideOutSuggestionDialog.<init>(DeleteBranchOnSlideOutSuggestionDialog.kt:27)
	at com.virtuslab.gitmachete.frontend.actions.base.BaseSlideOutBranchAction.suggestBranchDeletion(BaseSlideOutBranchAction.java:140)
	at com.virtuslab.gitmachete.frontend.actions.base.BaseSlideOutBranchAction.lambda$deleteBranchIfRequired$3(BaseSlideOutBranchAction.java:125)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:216)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:24)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:199)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:324)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:85)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:134)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:47)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:190)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:976)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:454)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:773)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:453)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:501)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

Reply:
Huh probably a dependency `implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"` will be needed once we change the build target finally to `2021.1`... just weird that it's not a problem when we use kotlin plugin `1.3.72` 🤔

Reply:
So no action required on this PR AFAICS